nebula_ros/schema/sub/hardware.json

```
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Hardware parameters.",
+  "type": "object",
+  "definitions": {
+    "launch_hw": {
+      "type": "boolean",
+      "readOnly": false,
+      "description": "Launch sensor hardware."
```

A more precise description would be preferable.
```suggestion
      "description": "Whether network sockets should be opened or not. If disabled, replay from NebulaPackets messages is enabled automatically."
```